### PR TITLE
Auto parse time and recent games

### PR DESCRIPTION
**Pull request by ChiefChippy2**
addition of recent games, accessible via Client.getRecentGames() or <Player>.getRecentGames()
added timestamp ( breaking change )

**Please describe changes**

BREAKING CHANGE - A lot of properties returning timestamp now doesn't exist or returns timestamps.
naming scheme : 
original name => new name ( date ) | new name ( timestamp )
firstJoin => firstJoinAt | firstJoin
mutedUntil => mutedUntil | mutedUntilTimestamp
timestamp => at | timestamp
createdAt => createdAt | createdAtTimestamp

if the property name is the same, new Date( _bla.createdAt_ ) can still work ( you can parse a date twice )

*Description*

- [x] I've added new features. (methods or parameters)
- [ ] I've fixed bug. (*optional* you can mention a issue if there is one)
- [ ] I've corrected the spelling in README, documentation, etc.
- [x] I've tested my code. (`npm run test`)

### CODE REVIEW

File: src/structures/Player.js [hunk 1]
```diff
@@ -62,6 +66,10 @@ class Player {
       arena: (data.stats.Arena ? new ArenaBrawl(data.stats.Arena) : null)
     } : null);
   }
+
+  getRecentGames () {
+    return this.getRecentGames(this.uuid, this);
+  }
```
Comment (StavZ) [inline src/structures/Player.js:72]: wut

**Review by StavZ — CHANGES_REQUESTED** [review]

File: src/structures/Player.js (hunk 1 above)
Comment (ChiefChippy2) [reply src/structures/Player.js:72]: so one can do 
```
Client.getPlayer("ign")
.then(player=>player.getRecentGames())
.then(recentgames=>{
//blabla
})
```

Comment (StavZ) [reply src/structures/Player.js:72]: image

Comment (ChiefChippy2) [reply src/structures/Player.js:72]: oh right I see

**Comment by ChiefChippy2** [comment]
changed last login in recent games to last logout ( makes more sense )
fixed issue ( thanks @StavZ )

**Review by StavZ — APPROVED** [review]

**Comment by StavZ** [comment]
image
Same error

**Pull request merged**